bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorage.java

```diff
@@ -335,4 +335,10 @@ public boolean isInForceGC() {
         return ledgerStorageList.stream()
             .map(single -> single.getGarbageCollectionStatus().get(0)).collect(Collectors.toList());
     }
+
+    @Override
+    public byte[] getEntriesOfLedger(long ledgerId) {
+        // TODO Auto-generated method stub
+        return null;
```

Comment:
not sure how can we deal with this

I guess we have to implement this one as well, or have some kind of protocol in which the bookie can report to the client something like 'I don't know which are the entries I have'

Reply:
An error code would suffice -- getEntriesOfLedger should therefore be able to raise a NotImplemented exception or whatever is most idiomatic.

Reply:
I would suggest throw UnsupportedOperationException for now.

Reply:
fixed it